### ISSUE TITLE
Build marked BROKEN with 404 file

### ISSUE THREAD

**Issue by lapo-luchini** [psa/libtensorflow1-freebsd-port#5]
Commit https://github.com/freebsd/freebsd-ports/commit/9a6115095e0e4cd57825421ef3324127a9c34043 has marked port as broken because «tries to fetch during the build».

Trying to reproduce locally, I get a:

`ERROR: Analysis of target '//tensorflow/tools/lib_package:libtensorflow.tar.gz' failed; build aborted: no such package '@zlib//': java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /usr/ports/work/usr/ports/science/libtensorflow1/work-default/bazel_out/ea330fc52515eb694a76ad53f73200df/external/zlib/zlib-1.2.11.tar.gz: GET returned 404 Not Found`

**Comment by lapo-luchini** [comment]
As reported in https://github.com/huo-ju/photoprism-freebsd-port/issues/13 it seems that [this patch](https://github.com/tensorflow/tensorflow/pull/32177/files) is solving the issue.

```diff
% cat files/patch-third__party_repo.bzl
--- third_party/repo.bzl.orig   2022-08-30 17:37:22 UTC
+++ third_party/repo.bzl
@@ -63,10 +63,7 @@ def _repos_are_siblings():
 # Apply a patch_file to the repository root directory
 # Runs 'git apply' on Unix, 'patch -p1' on Windows.
 def _apply_patch(ctx, patch_file):
-    if _is_windows(ctx):
-        patch_command = ["patch", "-p1", "-d", ctx.path("."), "-i", ctx.path(patch_file)]
-    else:
-        patch_command = ["git", "apply", "-v", ctx.path(patch_file)]
+    patch_command = ["patch", "-p1", "-d", ctx.path("."), "-i", ctx.path(patch_file)]
     cmd = _wrap_bash_cmd(ctx, patch_command)
     _execute_and_check_ret_code(ctx, cmd)
```

**Comment by psa** [comment]
Ah, I had that patch blocked behind a check for specific FreeBSD versions. I've removed the check and am doing a test build now.

**Comment by lapo-luchini** [comment]
BTW: I only tried it working on FreeBSD-13.1/amd64 myself (didn't try others).

**Comment by psa** [comment]
Build is clean. Fixed in [9dad684](https://github.com/psa/libtensorflow1-freebsd-port/commit/9dad68426c9e556b1ebe6e42ee9130d69ef17ebb)

**Comment by lapo-luchini** [comment]
BTW: no hurry, but I also opened a [FreeBSD PR](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=266130).
